api_v3/services/LiveStreamService.php

```diff
@@ -196,9 +196,13 @@ private function validateMaxStreamsNotReached(LiveEntry $liveEntry)
 		$maxPassthroughStreams = $liveEntryPartner->getMaxLiveStreamInputs();
 		KalturaLog::debug("Max Passthrough streams [$maxPassthroughStreams]");
 		$adminTagsOrigLimits = $liveEntryPartner->getMaxConcurrentLiveByAdminTag();
-		KalturaLog::debug('Current AdminTags: [' . $liveEntry->getAdminTags() . '] AdminTag limits : [' . print_r($adminTagsOrigLimits, true) . ']');
-		$adminTagsCounters = $adminTagsOrigLimits;
+		$isCloudTranscode = $this->isCloudTranscode($liveEntry->getConversionProfileId());
 		
+		$isAdminTagLimited = $this->isAdminTagLimited($liveEntry, array_keys($adminTagsOrigLimits));
+		if($isAdminTagLimited)
+		{
```

Comment:
better add comment: if has admin tag limit we don't going to check other limitation...